src/app/store.ts

```diff
@@ -18,4 +28,8 @@ export type AppThunk<ReturnType = void> = ThunkAction<
   unknown,
   Action<string>
 >;
+
+// function usersReducer(state: unknown, action: UnknownAction): unknown {
+//   throw new Error('Function not implemented.');
```

Comment:
It seems like you have a commented out error throw statement here. If this is not needed, consider removing it to keep your code clean.